slack-api-client/src/main/java/com/slack/api/methods/impl/TeamIdCache.java

```diff
@@ -32,15 +32,16 @@ public String lookupOrResolve(String token) {
                 FormBody.Builder form = toForm(AuthTestRequest.builder().token(newToken).build());
                 Response response = methodsImpl.runPostFormWithToken(form, Methods.AUTH_TEST, token);
                 AuthTestResponse authTest = methodsImpl.parseJsonResponseAndRunListeners(null, null, response, AuthTestResponse.class);
-                if (authTest.isOk()) {
+                if (authTest != null && authTest.isOk()) {
                     if (log.isDebugEnabled()) {
                         log.debug("Created cache for an auth.test API call (token: {}, team_id: {})",
                                 token.substring(0, 16) + "...", authTest.getTeamId());
                     }
                     // for org admin user's token, this value can be an enterprise_id
                     return authTest.getTeamId();
                 } else {
-                    log.error("Got an unsuccessful response from auth.test API (error: {})", authTest.getError());
+                    String error = authTest != null ? authTest.getError() : "";
```

Comment:
Perhaps in the case that `authTest` is null, we set a more descriptive error than the empty string? I know you said `authTest` cannot be null - but who knows how this code will change in the future 😃 

```suggestion
                    String error = authTest != null ? authTest.getError() : "AuthTestResponse is null";
```

Reply:
```suggestion
                    String error = authTest != null ? authTest.getError() : "(empty response)";
```

Reply:
Thanks! Perhaps, mentioning an empty response body would be easier to understand